test/README.md

```diff
@@ -0,0 +1,15 @@
+# Testing Directory README
+
+All files and configurations used for testing as detailed in the [Testing Guide](https://karpenter.sh/preview/testing-guide/) live here. Configurations and setup files here will be used in the future to test PRs.
+
+## File Directory
+- `/infrastructure`: Testing infrastructure
+- `/suites`: Tekton CRDs defining test suites
+
+## Testing Infrastructure Design Choices
+Testing infrastructure will be divided up into three layers: Management Cluster, Test Orchestration, and Clusters Under Test. The Management Cluster will be a Kubernetes cluster with configured add-ons to run a Test Orchestration tool that will create Clusters Under Test where Karpenter will be tested.
+- `Management Cluster`: An EKS Cluster with configured Add-ons and Tekton
```

Comment:
Can we link to Tekton for more information the first time it appears in the text?